static/app/components/charts/baseChart.tsx

```diff
@@ -556,6 +599,23 @@ function BaseChartUnwrapped({
     ]
   );
 
+  const coreOptions = useMemo(() => {
```

Comment:
This doesn't need to be a useMemo. The cost of initializing a useMemo is not worth it sometimes.

Reply:
Agree, I'm erroring on the side of safety here as I'm not sure how echarts deals with these and what updated they might trigger in the internal chart instance. My thinking here is that even if useMemo is excessive, it should still be a faster check to perform than dealing with any chart updates :/